terraform/deployments/apps/test/content-store/common.tf

```diff
@@ -0,0 +1 @@
+../common.tf
```

Comment:
Interesting idea to use symlinks for this. Let's see how that goes - seems like a win for now at least!

Looking at the bigger picture I think it's probably a smell that Terraform is not the right tool for config management of this complexity/scale, but let's not dwell on that 😅